br/pkg/storage/writer.go

```diff
@@ -16,6 +16,8 @@ import (
 // CompressType represents the type of compression.
 type CompressType uint8
 
+var WriteBuffersize = 64 * 1024
```

Comment:
seems not used

Reply:
uploaderWriter := newBufferedWriter(uploader, WriteBufferSize, NoCompression)